specs/user_spec.rb

```diff
@@ -58,5 +60,91 @@
         expect(trip.passenger.id).must_equal 9
       end
     end
+
+
+
+
+    it "sums total amount of money user has spent on their trips" do
+      # Arrange - Add two more trips to current user
+      trip = RideShare::Trip.new(id: 9, driver: nil, passenger: @user,
+                                 start_time: Time.parse("2016-08-08"),
+                                 end_time: Time.parse("2016-08-09"),
+                                 cost: 10,
+                                 rating: 5)
+      @user.add_trip(trip)
+
+
+      trip = RideShare::Trip.new(id: 10, driver: nil, passenger: @user,
+                                 start_time: Time.parse("2016-08-08"),
+                                 end_time: Time.parse("2016-08-09"),
+                                 cost: 15,
+                                 rating: 5)
+      @user.add_trip(trip)
+      # binding.pry
+      # Act - call net_expenditures method on user
+      passenger_expenditure = @user.net_expenditures
+
+      # Assert - state correct result
+      expect (passenger_expenditure).must_equal 30
+    end
+
+    it "Does not include trips in progress in calculation of total amount spent on trips" do
+
+      run_trip_dispatcher = RideShare::TripDispatcher.new()
+        trip2 = RideShare::Trip.new(id: 9, driver: nil, passenger: @user,
```

Comment:
Good work getting this edge case. Another one to consider: what if the user has not yet taken a trip?